ui/apps/platform/src/Components/CompoundSearchFilter/attributes/virtualMachine.ts

```diff
@@ -20,3 +20,17 @@ export const VirtualMachineComponentVersion: CompoundSearchFilterAttribute = {
     searchTerm: 'Component Version',
     inputType: 'text',
 };
+
+export const VirtualMachineID: CompoundSearchFilterAttribute = {
+    displayName: 'ID',
+    filterChipLabel: 'Virtual Machine ID',
+    searchTerm: 'Virtual Machine ID',
```

Comment:
Find in Files `filterChipLabel` suggests pattern for visible text (like column heading) is sentence case:
* Virtual machine ID
* Virtual machine name

Reply:
Whoops, updated